### ISSUE TITLE
Unable to get all monitoring metrics for num_undelivered_messages

### ISSUE THREAD

**Issue by samskillman** [GoogleCloudPlatform/google-cloud-python#3176]
I've written a simple utility to monitor a bunch of pubsub queues, in particular the number of undelivered messages. I'm currently unable to page through all the subscriptions and am instead only able to see 32 results. 
```
    client = monitoring.Client()
    q = client.query(
        "pubsub.googleapis.com/subscription/num_undelivered_messages",
        minutes=30
    )
    print len(list(q))  # Always results in 32, I have more than that.
```

I've also tried using the select_resource to filter the query, but that seems to only filter on the first 32 results. I'm using python2.7 and google-cloud-monitoring==0.23.0

**Comment by dhermes** [comment]
@samskillman I am trying to reproduce, any suggestions for reproducing? If you make the same query in the [APIs explorer][1] can you get more than 32?

[1]: https://developers.google.com/apis-explorer/#p/monitoring/v3/

**Comment by samskillman** [comment]
@dhermes thanks I went through https://developers.google.com/apis-explorer/#search/monitoring/cloudmonitoring/v2beta2/cloudmonitoring.timeseries.list and got the same result. I think my confusion came from a lag between when the subscription started processing messages and when the monitoring metrics showed up (which they now do as expected). I was used to the 3-5 minute lag between a new subcription and it showing up in both the stackdriver monitoring interface and through the API call in google-cloud-python, but this seemed to be more on > 30 minute lag. Probably can close the issue for now since it doesn't apply only to google-cloud-python, but the time delay between the creation of the subscription and it showing up through the monitoring API definitely caused quite a bit of confusion. Maybe worth a note somewhere in the docs if it is a reproducible/expected delay? I have used this api quite a bit and this was the first time I ran into the delay. 

Thanks again!
